packages/devtools/src/commands/elementSendKeys.js

```diff
@@ -1,4 +1,5 @@
 import { getStaleElementError } from '../utils'
+import path from "path"
```

Comment:
this is failing eslint tests
```suggestion
import path from 'path'
```